frontend/src/lib/getChartData.ts

```diff
@@ -10,79 +10,73 @@ const getChartData = ({
 	createDataset,
 }: GetChartDataProps): ChartData => {
 	const uniqueTimeLabels = new Set<number>();
-	queryData.forEach((data) => {
-		data.queryData.forEach((query) => {
-			query.values.forEach((value) => {
-				uniqueTimeLabels.add(value[0]);
-			});
-		});
-	});
-	const labels = Array.from(uniqueTimeLabels).sort((a, b) => a - b);
+	const allLabels: string[] = [];
```

Comment:
I think it will be helpful for anyone referring to this function in future, if a sample can be added on the transformation that this function is performimng, like a sample input and output.